packs/octopusdeploy/actions/add_machine.yaml

```diff
@@ -0,0 +1,27 @@
+---
+name: add_machine
+runner_type: python-script
+description: Add a new machine (tentacle) to an Octopus Environment
+enabled: true
+entry_point: add_machine.py
+parameters:
+  environment_id:
+    type: string
+    description: ID of the environment in Octopus to add this machine to
+    required: true
+  name:
+    type: string
+    description: The name of the server
+    required: true
+  uri:
+    type: string
+    description: Uri of the Octopus tentacle, typically https://host:10933/
+    required: true
+  thumbprint:
+    type: string
+    description: The encryption thumbprint generated by the installer
+    required: true
+  roles:
+    type: string
```

Comment:
We supported `array` type so you should use that. This way you don't need to use .split, etc. in the action.

For convenience, CLI allows you to specify `array` parameters as a comma delimited string (those get converted to an array internaly in the CLI) - http://docs.stackstorm.com/cli.html#specifying-parameters-which-type-is-array